dumpling-overview.md

```diff
@@ -347,7 +347,7 @@ Finally, all the exported data can be imported back to TiDB using [TiDB Lightnin
 | `--case-sensitive`           | whether table-filter is case-sensitive                                                                                                                                                                                                                                                                                             | false (case-insensitive)                   |
 | `-h` or `--host`             | The IP address of the connected database host                                                                                                                                                                                                                                                                                      | "127.0.0.1"                                |
 | `-t` or `--threads`          | The number of concurrent backup threads                                                                                                                                                                                                                                                                                            | 4                                          |
-| `-r` or `--rows`             | Divide the table into specified rows of data (generally applicable for concurrent operations of splitting a large table into multiple files.                                                                                                                                                                                       |
+| `-r` or `--rows`             | Split the table into a specified row of data (generally applicable for concurrent operations of splitting a large table into multiple files. When the upstream database is TiDB v3.0 or later versions, a value of this parameter greater than 0 indicates that the TiDB region information is used for splitting and the value specified here will no longer take effect.                                                                                                                                                                                 |
```

Comment:
"a specified row of data" is a littler unclear to me, how about "pieces of data with specified number of rows"